programs/mango-v4/tests/program_test/mango_client.rs

```diff
@@ -1503,6 +1503,49 @@ impl ClientInstruction for GroupCreateInstruction {
     }
 }
 
+pub struct GroupEditInstruction {
+    pub group: Pubkey,
+    pub admin: TestKeypair,
+    pub fees_mngo_bonus_rate: f32,
+    pub fees_mngo_token_index: TokenIndex,
+    pub fees_swap_mango_account: Pubkey,
+}
+#[async_trait::async_trait(?Send)]
+impl ClientInstruction for GroupEditInstruction {
+    type Accounts = mango_v4::accounts::GroupEdit;
+    type Instruction = mango_v4::instruction::GroupEdit;
+    async fn to_instruction(
+        &self,
+        _account_loader: impl ClientAccountLoader + 'async_trait,
+    ) -> (Self::Accounts, instruction::Instruction) {
+        let program_id = mango_v4::id();
+        let instruction = Self::Instruction {
+            admin_opt: None,
+            fast_listing_admin_opt: None,
+            security_admin_opt: None,
+            testing_opt: None,
+            version_opt: None,
+            deposit_limit_quote_opt: None,
```

Comment:
recommend making a function that returns a all-None edit instruction, like for perp_edit